docs/development-guide.md

```diff
@@ -108,7 +110,7 @@ Make sure your editor uses the environment you just created with Hatch. For Visu
 $ code .
 ```
 
-**NOTE:** The Nuxt image does not automatically refresh while running in development mode. Any changes will need a rebuild. This gets tired fast, so it's easier to run Nuxt outside Docker and call through to the `backend` for API calls. You can then view the frontend at `http://localhost:3000` and the backend api endpoints at `http://localhost/redoc`. This problem won't be a concern in production.
+**NOTE:** The React image does not automatically refresh while running in development mode. Any changes will need a rebuild. This gets tired fast, so it's easier to run React outside Docker and call through to the `backend` for API calls. You can then view the frontend at `http://localhost:3000` and the backend api endpoints at `http://localhost/redoc`. This problem won't be a concern in production.
```

Comment:
I'm not sure I like this, the app should be self-contained and shouldn't require a little workaround like this. Thoughts?

Reply:
So there is a solution to this problem; it would require us to change the Dockerfile of the frontend path to have a "dev" and "prod" execution steps. I think it's outside of the scope of work for now, but we can both file an issue ticket and note it here.